hephaestus_config.yaml

```diff
@@ -1,139 +1,102 @@
-# Hephaestus Configuration File
```

Comment:
Loved the reformatting of the file, but please keep the comments and don't use your personal path, keep it as `./your_project`